src/tribler-core/tribler_core/config/tribler_config.spec

```diff
@@ -106,3 +106,6 @@ history_size = integer(min=1, default=20)
 [popularity_community]
 enabled = boolean(default=True)
 cache_dir = string(default=health_cache)
+
+[error_handling]
+automatically_send_unhandled_exceptions=boolean(default=True)
```

Comment:
- Suggestion: it is basically representing user consent to send the reports so maybe could be renamed as `error_reporting_consent` or `user_consent`.
- I think the default should be false